$OpenBSD: patch-src_Resources_cc,v 1.1 2019/09/26 10:36:53 cwen Exp $

ports-gcc fix for:
error: changes meaning of 'list' from 'class std::__cxx11::list<_WaAction*>'

Index: src/Resources.cc
--- src/Resources.cc.orig
+++ src/Resources.cc
@@ -1568,7 +1568,7 @@ void ResourceHandler::LoadActions(WaScreen *wascreen) 
                                                            NULL);
                             str = str + i3 + 1;
                             ReadActions((char *) buffer2, defs, &wacts,
-                                        &ext_list->list, wascreen);
+                                        &ext_list->alist, wascreen);
                         }
                         else if (str[0] == 'n' && str[1] == '/') {
                             for (i3 = 2; str[i3] != '\0' &&
@@ -1583,7 +1583,7 @@ void ResourceHandler::LoadActions(WaScreen *wascreen) 
                                                            NULL);
                             str = str + i3 + 1;
                             ReadActions((char *) buffer2, defs, &wacts,
-                                        &ext_list->list, wascreen);
+                                        &ext_list->alist, wascreen);
                         }
                         else if (str[0] == 't' && str[1] == '/') {
                             for (i3 = 2; str[i3] != '\0' &&
@@ -1598,7 +1598,7 @@ void ResourceHandler::LoadActions(WaScreen *wascreen) 
                                                            str + 2);
                             str = str + i3 + 1;
                             ReadActions((char *) buffer2, defs, &wacts,
-                                        &ext_list->list, wascreen);
+                                        &ext_list->alist, wascreen);
                         }
                         else if (! strncasecmp(str, "window", 6)) {
                             str = str + 6;
